clvm/operators.py

```diff
@@ -43,22 +55,19 @@ class OperatorDict(dict):
 
     def __new__(class_, d: Dict):
         self = super(OperatorDict, class_).__new__(class_, d)
-        self.unknown_op_handler = self.unknown_op_raise
+        self.unknown_op_handler = default_unknown_op
         return self
 
     def __call__(self, op: bytes, arguments: CLVMObject) -> Tuple[int, CLVMObject]:
         f = self.get(op)
         if f is None:
-            def f(args):
-                return self.unknown_op_handler(op, args)
-        return f(arguments)
+            return self.unknown_op_handler(op, arguments)
+        else:
+            return f(arguments)
```

Comment:
(You don't really need the `else:` here. It's a taste thing, but my taste is to not have it since it consumes space, possibly adds an opcode to the python, and isn't particularly confusing.)

Reply:
yeah, I'm going back-and-forth. This way one can pretend that `if`s are expressions :)

Reply:
The return value of a `return` expression is None. :P

Reply:
you need to squint pretty hard :)